avm/res/network/application-gateway/CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 The latest version of the changelog can be found [here](https://github.com/Azure/bicep-registry-modules/blob/main/avm/res/network/application-gateway/CHANGELOG.md).
 
+## 0.7.3
```

Comment:
This should be a minor version change to e.g. `0.8.0`

Please update `version.json` and this file and also re-run `Set-AVMModule`